src/services/intervieweeService.js

```diff
@@ -28,9 +28,37 @@ exports.getInterviewee = async (intervieweeId) => {
 exports.getInterviewees = async (projectId) => {
   try {
     const { candidates } = await Project.findOne({ _id: projectId }).populate("candidates").lean();
-
+    
     return { candidates };
   } catch (error) {
     return { error };
   }
 };
+
+exports.updateInterviewee = async ({ intervieweeId, interviewee }) => {
+  try {
+    const intervieweeData = await Interviewee.findOne({ _id: intervieweeId });
+    
+    Object.keys(interviewee.filterScores).forEach((key) => {
+      if (!intervieweeData.filterScores[key]) {
+        intervieweeData.filterScores[key] = [];
+      }
+      
+      intervieweeData.filterScores[key].push(interviewee.filterScores[key]);
+    });
+    
+    const updatedInterviewee =  await Interviewee.findByIdAndUpdate(intervieweeId, {
+      $push: {
+        questions: { $each: interviewee.questions },
+        comments: { $each: interviewee.comments },
+      },
+      $set: {
+        filterScores: { ...intervieweeData.filterScores },
```

Comment:
위의 코드리뷰를 반영하면

```suggestion
        filterScores: { ...newFilterScores },
```

가 되면 되겠네요